### DISCUSSION TITLE
Not booting on Mac OS Big Sur and up on latest EFI

### DISCUSSION THREAD

**Discussion by ray0189** [badstorm/surface-pro-7-opencore#58] [General]
Hey guys is anyone els having issues where it's not booting on the latest EFI file as well as slow boots I tried putting the NVME fix kext in but that also freezes on the Apple logo or I get like a no entry sign can someone help me please.

**Comment by badstorm** [comment]
I only tested the last EFI folder  with Monteray. Try to update the [`AirportItlwm`](https://github.com/OpenIntelWireless/itlwm) with the Big Sure version or use [`itlwm`](https://github.com/OpenIntelWireless/itlwm)

**Reply by andremiclat** [reply]
does this efi work with sp5?
